README.md

```diff
@@ -1,21 +1,21 @@
-# PyCon TW Reviewer Guide 審稿指南
+# PyCon APAC 2022 Reviewer Guide 審稿指南
 
 (English below)
 
-您好，歡迎您加入 PyCon TW 審稿的行列。請點選 [審稿指南](reviewer-guide.md) 開始了解審稿流程。
+您好，歡迎您加入 PyCon APAC 2022 審稿的行列。請點選 [審稿指南](reviewer-guide.md) 開始了解審稿流程。
 
 本次審稿的表定時程如下：
-+ Apr 26: Call for Proposals 截止
-+ Apr 27 ~ May 17: 第一階段審稿
-+ May 24 ~ Jun 6: 投稿者修稿階段
-+ Jun 7 ~ Jun 20: 第二階段審稿
-+ Jun 27: 完成入選、備取、拒絕稿件清單
++ Apr. 15: Call for Proposals 截止
```

Comment:
nitpick: Not sure whether the `.` is needed. but either way should work

Reply:
Yeah I checked and seems either is ok, let me remove `.` then. Thanks!

Reply:
fix done.